src/pages/Projects.jsx

```diff
@@ -52,127 +27,194 @@ const linkStyle = {
   }
 };
 
-const projectGridLogo = (index, title, logo, links) => {
-  return (
-    <Grid item xs={12} md={6} order={{ xs: 0, md: index % 2 === 0 ? 2 : 1 }}>
-      {/* Logo */}
-      <CardMedia
-        component="img"
-        image={logo ?? null}
-        alt={`${title} logo`}
-        sx={{
-          mx: 'auto',
-          objectFit: 'contain',
-          maxWidth: '90%',
-          height: '150px'
-        }}
-      />
-      {/* Logo ends */}
-      {/* Links */}
-      <Stack direction="row" justifyContent="space-evenly" pt={{ xs: 0, sm: '1rem' }}>
-        {links.map(({ href, icon }) => renderingLinks(href, icon))}
-      </Stack>
-      {/* Links end */}
-    </Grid>
-  );
-};
+const boxStyle = (index) => ({
+  mb: { xs: 5, sm: 10 },
+  p: 2,
+  borderRadius: '25px',
+  background: {
+    md:
+      index % 2 === 0
+        ? 'linear-gradient(270deg, rgba(217, 217, 217, 0) 25.54%, rgba(217, 217, 217, 0.4) 54.29%)'
+        : 'linear-gradient(90deg, rgba(217, 217, 217, 0) 25.54%, rgba(217, 217, 217, 0.4) 54.29%)',
+    xs: 'linear-gradient(0deg, rgba(217, 217, 217, 0) 18.54%, rgba(217, 217, 217, 0.4) 82.29%)'
+  },
+  display: 'flex',
+  flexDirection: { xs: 'column', md: 'row' }
+});
+
+const ProjectTitle = ({ index, title, logo, links }) => {
+  const isSmallScreen = useMediaQuery('(max-width:500px)');
+  const iconSize = isSmallScreen ? '30' : '45';
 
-const projectGridContent = (index, description, title, status, techStack) => {
   return (
-    <Grid maxWidth={'xl'} item xs={12} md={6} order={{ xs: 0, md: index % 2 === 0 ? 1 : 2 }}>
-      <CardContent>
-        {/* Title and status */}
-        <Stack
-          direction={{ xs: 'column', sm: 'row' }}
-          spacing={1}
+    <Stack
+      minWidth={{ xs: '100%', md: '50%' }}
+      order={{ xs: 0, md: index % 2 === 0 ? 2 : 1 }}
+      spacing={{ xs: 1, md: 2 }}
+      alignItems="center"
+    >
+      {logo && (
+        <CardMedia
+          component="img"
+          image={logo}
+          alt={title ? `${title} logo` : 'project logo'}
           sx={{
-            alignItems: 'center',
-            justifyContent: 'space-between',
-            mb: '2rem'
+            mx: 'auto',
+            objectFit: 'fill',
+            maxWidth: '90%',
+            height: '150px'
```

Comment:
pass logo gets squashed because of height constraint. changing line 66 to ` height: logo === '/assets/PassLogo.png' ? 'auto' : '150px'` should be sufficient. I know it throws off the padding but we can just call that good enough for now unless you want to mess with that too.
![Screenshot 2023-12-27 at 11 19 26 AM](https://github.com/codeforpdx/codepdx_website/assets/114430929/cd61d56e-b6c5-465a-b394-74e16b1cb5b0)